scipy/stats/_stats_py.py

```diff
@@ -10185,14 +10189,227 @@ def quantile_test(x, *, q=0, p=0.5, alternative='two-sided'):
 #####################################
 
 
+def wasserstein_distance_nd(u_values, v_values, u_weights=None, v_weights=None):
+    r"""
+    Compute the Wasserstein-1 distance between two 2D discrete distributions.
```

Comment:
```suggestion
    Compute the Wasserstein-1 distance between two N-D discrete distributions.
```
?

The _arrays_ are 2D; that second dimension of the array is used to represent an arbitrary number of dimensions of the distribution. That's why it's called `wasserstein_distance_nd`, after all.